assets/js/base/context/providers/cart-checkout/payment-methods/reducer.ts

```diff
@@ -32,17 +32,14 @@ const reducer = (
 ): PaymentMethodDataContextState => {
 	switch ( type ) {
 		case STATUS.STARTED:
-			return state.currentStatus !== STATUS.STARTED
```

Comment:
I removed this to fix a side effect of using status started for express methods. If saved methods are used, started status is already set. Therefore, clicking an express method will not clear any payment method data. This change allows it to clear paymentMethodData.

Reply:
There was a test for this FYI

Reply:
I think it was that started status immediately changed back to pristine in the previous version, whereas now we need it to stay 'started'.